### PR TITLE
Study plan component

### PR DESCRIPTION
**Pull request by tonifisler**
Waiting on the refactoring before fixing and merging this!

CSS GRID POWERED!

### CODE REVIEW

**Comment by Yago** [comment]
The file structure is wrong, please copy all your work inside a fresh new branch/PR from `dev` to avoid conflicts and regressions

File: assets/components/content-types/laboratory/laboratory.scss [hunk 1]
```diff
@@ -1 +1,286 @@
...
+    font-size: 0.6875 * $font-size-base;
+
+    > div {
+      border-top: $table-border-width solid $table-border-color;
+      border-bottom-width: $table-border-width * 2;
+    }
+  }
+
+  .line-up {
+    position: sticky;
```
Comment (knuch) [inline assets/components/content-types/laboratory/laboratory.scss]: couldn't see any effect of this sticky declaration. Is that intended?

File: assets/components/content-types/study-plan/study-plan-ie.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+@charset "utf-8";
+
+/* stylelint-disable */
+// This file is intended for IE 11 and has autoprefixer and stylelint off.
+
+// IE 11 table headings
+@include ie11('.table-like .cours-title') {-ms-grid-column: 2;}
```
Comment (knuch) [inline assets/components/content-types/study-plan/study-plan-ie.scss]: The result on IE11 - Windows 8.1 (browserstack):
capture d ecran 2018-08-06 a 13 39 07

File: assets/components/content-types/laboratory/laboratory.scss [hunk 3]
```diff
@@ -1 +1,286 @@
...
+    top: 0;
+    background-color: $table-head-bg;
+  }
+
+  // LOURD
+  @supports (display: contents) {
+    display: grid;
+    grid-template-columns: var(--table-columns);
```
Comment (knuch) [inline assets/components/content-types/laboratory/laboratory.scss]: Nice! 🎉✨

File: assets/components/content-types/study-plan/study-plan-ie.scss (hunk 2 above)
Comment (tonifisler) [reply assets/components/content-types/study-plan/study-plan-ie.scss]: Ah shit... I tested on Windows 7, I think it looked better than this :(

File: assets/components/content-types/laboratory/laboratory.scss (hunk 1 above)
Comment (tonifisler) [reply assets/components/content-types/laboratory/laboratory.scss]: Yes the table head sticks on supporting browsers.

File: assets/components/content-types/laboratory/laboratory.scss [hunk 4]
```diff
@@ -1 +1,286 @@
 @charset 'utf-8';
+
```
Comment (Yago) [inline assets/components/content-types/laboratory/laboratory.scss]: It's the wrong file 😂

**Comment by Yago** [comment]
Huge and great job @tonifisler ! 👏

**Pull request merged**